docs/userguide/declarative_config.rst

```diff
@@ -192,28 +187,29 @@ platforms                       platform           list-comma
 provides                                           list-comma
 requires                                           list-comma
 obsoletes                                          list-comma
-==============================  =================  =================  =============== =====
+==============================  =================  =================  =============== ==========
 
-.. note::
-    A version loaded using the ``file:`` directive must comply with PEP 440.
-    It is easy to accidentally put something other than a valid version
-    string in such a file, so validation is stricter in this case.
+**Notes**:
+
+.. [#meta-1] The ``version`` file attribute has only been supported since 39.2.0.
+
+   A version loaded using the ``file:`` directive must comply with PEP 440.
+   It is easy to accidentally put something other than a valid version
+   string in such a file, so validation is stricter in this case.
 
-Notes:
-1. The ``version`` file attribute has only been supported since 39.2.0.
 
 Options
 -------
 
-=======================  ===================================  =============== =====
+=======================  ===================================  =============== =========
 Key                      Type                                 Minimum Version Notes
-=======================  ===================================  =============== =====
+=======================  ===================================  =============== =========
 zip_safe                 bool
 setup_requires           list-semi
 install_requires         list-semi
-extras_require           section
+extras_require           section                                              [#opt-2]_
 python_requires          str
-entry_points             file:, section
+entry_points             file:, section                       51.0.0
```

Comment:
Why is this 51? I thought I've been using this for years, well before 51 was even released? I could have sworn it at least works on 44. There's nothing in the 51 release notes that mentions this either.

Reply:
Was this supposed to be 41 instead? That sounds about right.